modules/lang-painless/src/main/resources/org/elasticsearch/painless/spi/java.util.txt

```diff
@@ -574,8 +574,8 @@ class java.util.Collections {
   Comparator reverseOrder()
   Comparator reverseOrder(Comparator)
   void rotate(List,int)
-  void shuffle(List)
-  void shuffle(List,Random)
+  void shuffle(List) @nondeterministic
+  void shuffle(List,Random) @nondeterministic
```

Comment:
maybe that one is ok since it takes an explicit random?

Reply:
Agreed. Reverted.